File: CabService/src/test/java/co/inteleap/course/RideTest.java
package co.inteleap.course;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

class RideTest {

	@Test
	public void shouldCalculateFareForGivenTime() {
		Ride ride=new Ride(0,1);
		assertEquals(1,ride.getTotalFare());
	}
	
	@Test
	public void shouldCalculateFareForGivenDistance() {
		Ride ride=new Ride(1,0);
		assertEquals(10,ride.getTotalFare());
	}
	
	@Test
	public void shouldCalculateFareForGivenTimeAndDistance() {
		Ride ride=new Ride(0,1);
		assertEquals(1,ride.getTotalFare());
	}

}

File: TDD-Kotlin/src/test/kotlin/RideTest.kt
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class RideTest {
    @Test
    fun shouldCalculateForAGivenTime(){
        val ride=Ride(1.0,0.0)
        assertEquals(1.0,ride.Fare())
    }
    @Test
    fun shouldCalculateForAGivenTimeAndForAGivenDistance(){
        val ride=Ride(1.0,1.0)
        assertEquals(11.0,ride.Fare())
    }
}